compiler-rt/lib/scudo/standalone/flags.cpp

```diff
@@ -68,6 +69,13 @@ void initFlags() {
   Parser.parseString(getCompileDefinitionScudoDefaultOptions());
   Parser.parseString(getScudoDefaultOptions());
   Parser.parseString(getEnv("SCUDO_OPTIONS"));
+  if (const char *V = getEnv("SCUDO_ALLOCATION_RING_BUFFER_SIZE")) {
```

Comment:
Could you replace this with FlagsParser::runHandler("SCUDO_ALLOCATOR_RING_BUFFER_SIZE", V)? I know it's not a public function, but maybe make a version that is public, one that doesn't return a bool and handles the error internally?

I think you should still do some kind of verification of this parameter because it makes me nervous if you plan to expose this, we should probably add more verification to all of the flags read in the flags parser code.